src/main/webapp/components/results/results.js

```diff
@@ -322,7 +347,9 @@ const Container = props => {
     { error, loading, refetch },
   ] = useUserPrefs()
 
-  if (loading) {
+  const aliasResults = useQuery(query)
```

Comment:
✏️ Could also do `const { data: aliasData, loading: aliasLoading, error: aliasError } = useQuery(query)` or something like that to avoid doing `aliasResults.loading` `aliasResults.error` and `aliasResults.data`

Reply:
Changed to rename on destructure